libafl/src/state/mod.rs

```diff
@@ -828,9 +828,10 @@ where
         if forced {
             let _: CorpusId = fuzzer.add_input(self, executor, manager, input)?;
         } else {
-            let (res, _) = fuzzer.evaluate_input(self, executor, manager, input)?;
+            let (res, _) = fuzzer.evaluate_input(self, executor, manager, input.clone())?;
```

Comment:
do we need the clone here? It's not a super hot path but still wondering if there's some way, like returning the input again or so

Reply:
The nested call-stack makes it difficult to return input without breaking changes to ``evaluate_input_events``, ``process_execution``,   and ``evaluate_input_with_observers`` etc.

 I think the cleanest would be to make ``must_load_initial_inputs`` mandatory for ``State`` and in process_execution have `` if state is load_inital then fuzzer.add_disabled() else return``

Reply:
@domenukk unfortunately I think we do need a clone there, otherwise we'd be cloning down the line in ``process_execution`` when returning the input. I wasn't able to come up with a clean way to include ``must_load_initial_inputs`` in ``process_execution`` either.